src/worker.js

```diff
@@ -66,35 +75,45 @@ const initServer = (app, config, callback) => {
 
       // In a promise in case one of the plugins throws an error.
       new Promise(() => { // eslint-disable-line no-new
-        const manager = new BatchManager(req, res, req.body, config);
-        errorSync(err, config.plugins, manager);
+        const manager = new BatchManager(req, res, req.body, this.config);
+        errorSync(err, this.config.plugins, manager);
       });
 
       return;
     }
-    shutDownSequence(err, req, 1);
+    this.shutDownSequence(err, req, 1);
+  }
+
+  initialize() {
+    // run through the initialize methods of any plugins that define them
+    runAppLifecycle('initialize', this.config.plugins, this.config)
```

Comment:
should this be `return`ed?

Reply:
It's not in the original `initServer`.  I think returning this would be a bit misleading, as it would be a promise around the creation of a server, when what is most likely to be expected would be the server listening (which triggers `callback` at some point later)